ReactiveXComponent/Common/StateMachineInstance.cs

```diff
@@ -7,9 +7,9 @@ public class StateMachineInstance
     {
         public long StateMachineId { get; set; }
 
-        public long AgentId { get; set; }
```

Comment:
These properties' types should be consistent all over the code. XComponent expects this one to be a _long_ for example.